networking/macsec/sanity_check/runtest.sh

```diff
@@ -0,0 +1,506 @@
+#!/bin/bash
+#
+# Copyright (c) 2013 Red Hat, Inc. All rights reserved.
+#
+# This copyrighted material is made available to anyone wishing
+# to use, modify, copy, or redistribute it subject to the terms
+# and conditions of the GNU General Public License version 2.
+#
+# This program is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+#
+
+FILE=$(readlink -f $BASH_SOURCE)
+NAME=$(basename $FILE)
+CDIR=$(dirname $FILE)
+RPATH=${RELATIVE_PATH:-"networking/macsec/sanity_check"}
+
+# Include Beaker environment
+source ${CDIR%/$RPATH}/networking/common/include.sh || exit 1
+
+KEY_0=81818181818181818181818181816161
+KEY_1=81818181818181818181818181816262
+NODE=1
+PEER=2
+SCI_NODE=0100567${NODE}12005452
+SCI_PEER=0100567${PEER}12005452
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun -l "modinfo macsec"
+        rlRun "modprobe macsec"
+        rlRun -l "ip macsec help" 255
+        if [ $? -eq 1 ]; then exit 1; fi
```

Comment:
Is this considered a test setup failure, do we need to abort in this case?

Reply:
Yes, abort it.
```patch
-        if [ $? -eq 1 ]; then exit 1; fi
+        if (( $? != 0 )); then
+                rlLog "Aborting test because 'ip macsec' not found"
+                rhts-report-result "$TEST" ABORTED "$OUTPUTFILE"
+                exit 1
+        fi
```